index.js

```diff
@@ -110,10 +120,39 @@ export const RepositoryCollection = {
   }
 }
 
+
 export const Repository = {
   self({ self, parent, source }) {
     return self || parent.ref.pop().pop().push('one', { name: source.name });
   },
+  issueOpened: {
+    async subscribe({ self }) {
+      const { name: owner } = self.match(root.users.one);
+      const { name: repo } = self.match(root.users.one.repos.one);
+      
+      await program.setTimer(`${owner}/${repo}`, 0, 10);
```

Comment:
This should trigger the timer immediately instead of 10s in the future. Call `timer(`${owner}/${repo}`)` directly?